ui/pages/swaps/prepare-swap-page/prepare-swap-page.js

```diff
@@ -796,6 +798,11 @@ export default function PrepareSwapPage({
     isTokenEligibleForMaxBalance &&
     hasPositiveFromTokenBalance;
 
+  const isNonDefaultToToken = !isSwapsDefaultTokenSymbol(
+    selectedToToken.symbol,
+    chainId,
+  );
```

Comment:
Would it be safer to compare addresses? Symbols can be spoofed

Reply:
Very good point. In this case I matched the way it was done a few lines above for the "from" token, but both should probably use address comparison instead. I'll make that change.

Reply:
Good catch, yes it would be

Reply:
Done. Changes made 👍